change/@fluentui-react-field-e888af0f-e684-4f66-8f86-8c950a9bb41e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Patch will trigger `9.0.0` release, it is okay of do you want to stay in alpha for now (i.e. use `prerelease`)?

Reply:
I've changed it to `prerelease`. Not sure why `yarn change` didn't default to that for an alpha package...

Reply:
Ahh it looks like I need to change the `disallowedChangeTypes` to allow prerelease and disallow patch. (I wonder if that should be the default configuration from the create-package script?)